### PR TITLE
add hover text for PV UV  -- optimized

### PR DESCRIPTION
**Pull request by xu-song**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): #108 

## What is the new behavior?

Add hover text for some confusing icon. In the above case , someone maybe be confusing with two date.
Hover text, such as “created” “modified” maybe more favorable in many cases.

* Screens with this changes: 

**page_pv with title**

image

image

 **page_pv with hover text.**, need the following config.
```
post_meta:
  item_text: false
```
image 

**site_pv with hover text:**
image

* Link to demo site with this changes: http://xusong.vip/2018/02/09/demo/

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by sli1989** [comment]
1. use the underscore character  `_` maybe not artistic. So using `title="{{ __('post.page_pv_title') }}"` can avoid the cutoff space in title test. 
1.   `visitors` maybe more appropriate than `page view` in `page_pv_header` to keep consistent with `leancloud_visitors` or just use one of them.
1. the `symbol.colon` or `:` should be considered to keep consistent with other post meta.

**Comment by xu-song** [comment]
thanks for your suggestion

**Comment by sli1989** [comment]
The hover text for `Visitors` is missing in demo?
3rd: add symbol `:` like `:` in `Symbols count in article:` or `≈` in `Reading time ≈`;
4th: i added `site_uv`, `site_pv`, and `page_pv` in post catalog. Maybe your PR is more appropriate.

but less modifies can act same 😂. 
wishes.

**Comment by ivan-nginx** [comment]
Why white paper in icon near the post? MB to set some icon here?

**Comment by xu-song** [comment]
Thanks for all your suggestion.

**Comment by ivan-nginx** [comment]
I just refactor this pull. Please, test it.

File: languages/zh-CN.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@ page:
 footer:
   powered: "由 %s 强力驱动"
   theme: 主题
+  site_pv: 总访问量
```
Comment (wafer-li) [inline languages/zh-CN.yml]: @ivan-nginx  

The field name is not correct.

It should be `total_views` as the same with `en.yml`

And the `site_pv` should be `total_visitors`

Comment (tsanie) [reply languages/zh-CN.yml]: @wafer-li seems no need to manual change the `zh-CN.yml`, any language res files will be fixed by the crowdin.

Comment (wafer-li) [reply languages/zh-CN.yml]: OK, then leave it to crowdin.

**Pull request merged**